### DISCUSSION TITLE
md5sum and sha256

### DISCUSSION THREAD

**Discussion by vermaete** [DOCGroup/ACE_TAO#1813] [Ideas]
Hi Johnny,

In the md5 file of the releases, the full path to the file is present.
`d1656851619aff15365270ccf2d56c6e  /home/johnny/DOC_ROOT/package-9846/ACE.tar.bz2`

IMHO, it's more common to add the path from where the md5 file is located.

Could it be useful to have the sha256 there too?

br,

**Comment by jwillemsen** [comment]
Feel free to open a PR to change the md5 generation and add sha256, see https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/bin/make_release.py for our release script and https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/docs/bczar/bczar.html for the release instructions

**Reply by vermaete** [reply]
@jwillemsen 

See: https://github.com/DOCGroup/ACE_TAO/pull/1819

I've tested with Ubuntu 20.04.  The md5 and and sha256 seem to be correct.
A `md5sum --check` and `sha256sum--check` did pass.

However, I did not run the the full pipeline at github.com.

Br,
